### ISSUE TITLE
[Research]: datatype/validity check of fields in schemas

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#65]
## User story
1. As a {HR-employee}
2. I want / need {to be sure that the person who provides information (defined by a schema) provides VALID information}
3. So that {I don't have to check manually if the field phone number is really filled with a phone number}

## Acceptance criteria
* check  if Lissi-API allows for an (automatic) check of the datatype of an attribute in a schema definition
* goal: be able to check if user provided phone number (digits) for the phone number field and not something else

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
